<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QTabWidget" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab_song">
   <attribute name="title">
    <string>管理乐谱</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_importSong">
          <property name="text">
           <string>导入新乐谱</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="toolTip">
               <string>歌曲名</string>
              </property>
              <property name="text">
               <string>歌名</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="s_name_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="toolTip">
               <string>作曲、编曲、演唱、后期...</string>
              </property>
              <property name="text">
               <string>作者</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="s_composer_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="toolTip">
               <string>歌曲的调性</string>
              </property>
              <property name="text">
               <string>调性</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="s_key_comboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="toolTip">
               <string>总谱、分谱或曲谱历史</string>
              </property>
              <property name="text">
               <string>版本</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="s_version_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="toolTip">
               <string>可选择PDF文档形式或PNG图片形式</string>
              </property>
              <property name="text">
               <string>类型</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="s_type_comboBox">
              <item>
               <property name="text">
                <string>PDF</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PNG</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,0,0">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="toolTip">
               <string>选择曲谱文件</string>
              </property>
              <property name="text">
               <string>文件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fPathlineEdit">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fPathpushButton">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="toolTip">
               <string>双击列表中的文件可以删除</string>
              </property>
              <property name="text">
               <string>文件
列表</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="file_listView"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="toolTip">
               <string>所属分类</string>
              </property>
              <property name="text">
               <string>分类</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="c_id_comboBox">
              <item>
               <property name="text">
                <string>无</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item alignment="Qt::AlignmentFlag::AlignTop">
             <widget class="QLabel" name="label_19">
              <property name="toolTip">
               <string>写点什么...</string>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::LeftToRight</enum>
              </property>
              <property name="text">
               <string>备注</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="s_remark_textEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QPushButton" name="pushButton_savesong">
            <property name="text">
             <string>保存修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_delsong">
            <property name="text">
             <string>删除乐谱</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_category">
   <attribute name="title">
    <string>管理分类</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_newcat">
          <property name="text">
           <string>添加新分类</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="category_listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>类名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="catename_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item alignment="Qt::AlignmentFlag::AlignTop">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>备注</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="cateremark_textEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_savecat">
            <property name="text">
             <string>保存修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_delcat">
            <property name="text">
             <string>删除分类</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
